pkg/quotes/README.md

```diff
@@ -39,14 +40,33 @@ last_price = price
 
 ## Index Price
 
-Used mainly in risk management for portfolio evaluation:
+Used mainly in risk management for portfolio evaluation, and is aggregated from multiple sources.
 
 ```
-index_price = EMA20(price x ( trade_size x weight / active_weights ))
-# active_weight being the sum of weight where this market exists (ex: KuCoin:5 + uniswap:50)
-# EMA20 is likely 20 at 1 min scale
+index_price = priceWeightEMA/weightEMA
+
+priceWeightEMA = EMA20((Volume)*(Price)*(DriverWeight/activeWeights))
```

Comment:
lets also provide life real example of calculations please

Reply:
yes, please, I'm a bit confused about how we are having one driverWeight divided by the sum of activeWeights.
How is the driver selected if it's the last 20 trades for priceWeightEMA from all sources? 
Or if it's only from one source, how is the end index price calculated from the results for each source?